app/views/organizations/staff/pets/show.html.erb

```diff
@@ -1,5 +1,13 @@
 <%= render DashboardPageComponent.new(crumb: :dashboard_pet, crumb_options: [@pet]) do |c| %>
-  <% c.with_header_title { @pet.name } %>
+  <% c.with_header_title do %>
```

Comment:
@Aaryanpal we just made a new pet avatar component, actually. Could you please use this here instead?

app/components/pet_avatar_component.rb

Reply:
okay

Reply:
[Screencast from 2024-08-28 20-06-53.webm](https://github.com/user-attachments/assets/672b1140-7df1-4606-ba5a-68363a7be735)

@kasugaijin, is it okay to show the initial letter of the pet's name when an image is not available? Also, should we update the pets index and fix the images to ensure they appear rounded?

![image](https://github.com/user-attachments/assets/7b28a017-015a-4fcd-9814-e379084cd857)